modules/apps/frontend-data-set/frontend-data-set-api/src/main/java/com/liferay/frontend/data/set/filter/BaseClientExtensionFDSFilter.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.frontend.data.set.filter;
+
+/**
+ * @author Iván Zaera Avellón
+ */
+public abstract class BaseClientExtensionFDSFilter implements FDSFilter {
+
+	public abstract String getModuleURL();
+
+	@Override
+	public String getType() {
+		return "clientExtension";
+	}
+
+}
```

Comment:
I don't see an immediate use case for this, i.e., allowing CX in java-based FDS config machinery. But I welcome the change anyways as it puts the CX at the same level as the rest of filters, as we did in the FDS itself :clap:

Reply:
Yes. I'm not sure if anyone will use it, but it makes a lot of sense and it's nice to have it.